common/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,35 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can you add a note why this was needed? Because as far as I know the one in common `main` dir should be being used already

Reply:
I would if I could remember.  I'm going to leave for now since it doesn't have anything in it.